src/devicecontroller/DefaultDeviceController.ts

```diff
@@ -356,18 +356,8 @@ export default class DefaultDeviceController
 
     // Update device and stream
     this.chosenVideoTransformDevice = device;
-    const newMediaStream = this.activeDevices['video'].stream;
     this.logger.info('video transform device uses previous stream');
 
-    // Input is not a MediaStream. Update constraints
-    if (!(inner as MediaStream).id) {
```

Comment:
I don't see any change in the device controller unit tests. Do we have any tests for this?

Reply:
The if statement is covered by the standard video transform test and the else is covered by this test https://github.com/aws/amazon-chime-sdk-js/blob/main/test/devicecontroller/DefaultDeviceController.test.ts#L1294. It is really direct related to this code block so I decided not to remove it.